packages/experimental/Shadow/src/__tests__/Shadow.test.tsx

```diff
@@ -54,12 +54,18 @@ const TestShadowOnChildViewWithProps: React.FunctionComponent<ShadowOnChildViewW
   );
 };
 
+jest.mock('@fluentui-react-native/experimental-appearance-additions', () => ({
+  useUserInterfaceLevel: jest.fn(),
+  useHorizontalSizeClass: jest.fn(),
+}));
+
```

Comment:
I don't know why - but this mock just wouldn't work unless I put it outside the describe block. I tried putting it in the beforeAll section, also tried putting it directly into each test. The only way it worked was when I took it out.

Reply:
Also not sure why - I first tried to do useUserInterfaceLevel: 'base' but got an error saying that the userInterfaceLevel call wasn't a function. Will try making the jest.fn() return 'base' but doesn't seem to matter for this test